package/python-kiwi-spec-template

```diff
@@ -231,14 +273,13 @@ outside of the scope of kiwi appliance building.
 %ifarch %ix86 x86_64
 %package -n kiwi-pxeboot
 Summary:        KIWI - PXE boot structure
+Group:          System/Management
```

Comment:
You don't need this, as you get it from the main source package. Same goes for `License` in all subpackages.